socs/agents/wiregrid_tiltsensor/agent.py

```diff
@@ -99,8 +112,25 @@ def acq(self, session, params=None):
                 # data taking
                 current_time = time.time()
                 msg, angles = self.tiltsensor.get_angle()
-                if self.sensor_type == 'sherborne':
-                    msg, temperatures = self.tiltsensor.get_temp()
+                self.log.info(f'self.tiltsensor.get_angle(): {msg}')
+                if angles[0] == -999:
+                    msg = 'Failed to get angle. -> Reconnect to the sensor'
+                    self.log.warn(msg)
+                    del self.tiltsensor
+                    self._connect()
+                    self.pm.sleep()  # DAQ interval
+                    continue
+                # No implementation of `get_temp()` now and ignore the below lines
+                # if self.sensor_type == 'sherborne':
```

Comment:
Are we just removing support for the 'sherbone' sensor type? Can you comment more about this removal?